sumobot.ino

```diff
@@ -59,3 +61,14 @@ bool inRange() {
     return false;
   }
 }
+
+void fullForward(){
+  Serial.print("GOOOooOOOoOOOoOOOOOOOo!!");
+  digitalWrite(rightWheelPin, HIGH);
```

Comment:
digitalWrite will not work in this case because the Servos actually use PWM signals to move. There's a library that we can use to generate these signals from Adafruit. Let's look that up.